iset.mm

```diff
@@ -41635,6 +41635,22 @@ arbitrary sets or real numbers will not be provable (without the law of
       ( cvv wcel cres resexg ax-mp ) ADEABFDECABDGH $.
   $}
 
+  $( Class restriction distributes over intersection.  (Contributed by FL,
+     6-Oct-2008.) $)
+  resindm $p |- ( Rel A -> ( A |` ( B i^i dom A ) ) = ( A |` B ) ) $=
+    ( wrel cres cdm cin resdm resindi incom inres inidm reseq1i 3eqtrri 3eqtr4g
+    ineq2d ) ACZABDZAAEZDZFQAFZABRFDQPSAQAGOABRHTAQFAAFZBDQQAIAABJUAABAKLMN $.
+
+  $( Restricting a function to a domain without one element of the domain of
+     the function, and adding a pair of this element and the function value of
+     the element results in the function itself.  (Contributed by AV,
+     2-Dec-2018.) $)
+  resdmdfsn $p |- ( Rel R -> ( R |` ( _V \ { X } ) )
+                             = ( R |` ( dom R \ { X } ) ) ) $=
+    ( wrel cdm csn cdif cres cvv cin indif1 incom eqtri difeq1i reseq2i resindm
```

Comment:
The comment does not match the statement. Ping also @avekens

Reply:
OK, I will change this in set.mm to "Restricting a relation to its domain without one element of the domain is
     the same as restricting the relation to the universe without this element.".

Reply:
I can't see: is there a $e-hypothesis saying that X e. dom R ? If yes, that's fine; if no, just remove "of the domain".

Reply:
OK, "of the domain" can be removed.

Reply:
I'll put this wording ("Restricting a relation to its domain without one element is the same as restricting the relation to the universe without this element.") into this pull request unless there is some reason you'd rather do so yourself.

Reply:
Hmm, on second thought I might be better off letting you make these changes to set.mm in #2475 and I can grab the wording from that branch for iset.mm. I'll try to do that within the next few hours.

Reply:
Yes, I changed it already in set.mm with #2475, so please do not change it (again) to avoid merge conflicts.

Reply:
Yeah, if we both make the exact same change git will just merge it without a conflict. If I make an almost-same change we'd need to resolve conflicts (which isn't the end of the world - at least if I do the resolution I'm not super scared of that task - but to make it even easier I'll let you make the changes in set.mm and I'll try to carry those over into iset.mm).